src/index.html

```diff
@@ -17,11 +17,466 @@
+      href="styles/style.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="logotype"
+          class="logo"
+        />
+      </a>
+      <nav class="nav_list">
+        <ul>
+          <li class="is-active">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Photo
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div
+      class="product"
+      data-qa="card"
+    >
+          <div class="product__picture product__picture--modifier">
+          </div>
+
+          <div class="product__name product__name--modifier">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="product__code product__code--modifier">
+            Product code: 195434
+          </div>
+
+          <div class="product__rating">
+            <div class="product__rating-stars">
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+              <div class="product__rating-stars-star"></div>
+            </div>
+            <div class="product__rating-reviews">Reviews: 5</div>
+          </div>
+
+          <div class="product__price product__price--modifier">
+            <div class="product__price-text">Price:</div>
+            <div class="product__price-digital">$2,199</div>
+          </div>
+          <div
+            class="product__button"
+          >
+            <label for="buy"
+            data-qa="card-hover">
```

Comment:
The `label` element is used here, but it is not semantically correct to wrap a `form` element with a `label`. Instead, the `label` should be associated with a specific form control using the `for` attribute.